apps/teams-test-app/e2e-test-data/secondaryBrowser.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "SecondaryBrowser",
+  "version": ">=2.12.0",
```

Comment:
Question: if the version changes here, do we also need to make sure the version matches in https://office.visualstudio.com/ISS/_git/metaos-hub-sdk-iOS/pullrequest/2273698?

Reply:
We don't have to. Version number is to tell back combat tool when this data should be taken for building E2E test. The update is to make sure that when it is the time to build data, it can process and build files properly leveraging on callAndResponse API as base.